Index.html

```diff
@@ -128,24 +174,31 @@ <h2>About me</h2>
             </ul>
            </div> 
         </div>
-
-        <div class="form-section">
+    </section>    
+    <section class="form">
+        <div id="form-text-container">
             <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form action="https://formspree.io/f/mrgdnqrz" method="post">
-                <input type="text" class="fill-area" name="full-name" required maxlength="25" placeholder="Full Name">
-                <input type="email" class="fill-area" name="email" required placeholder="Email Address">
-                <textarea rows="8" name="message" required maxlength="500" placeholder="Enter text here"></textarea>
-                <input type="submit" class="button-1" id="submit" value="Get in touch">
-            </form>
-            <div class="social-media">
-                <input type="image"  src="twitter.png">
-                <input type="image"  src="Linkedin icon.png">
-                <input type="image"  src="medium.png">
-                <input type="image"  src="github.png">
-                <input type="image"  src="angellist.png"><br>
-            </div>
-            <p id="email">alwynsimiyu@gmail.com</p>
         </div>
-    </div> 
+        <div id="form-container">
+            <form class="form-entries">
```

Comment:
- [ ] Please style the form section of your page to match the figma template.
### Your design
![1](https://user-images.githubusercontent.com/94551484/206829784-a1c86e6e-f252-4180-9d81-a6cbf58c20fa.png)


### Figma template
![f1](https://user-images.githubusercontent.com/94551484/206829787-d7c01ac4-6697-4631-9d6c-85f731780f50.png)

Reply:
Thank you. I have removed the flex-wrap property that was responsible for the issue. The form now closely mimics the template in positioning